app/views/space_events/_sidebar_events_simple.html.haml

```diff
@@ -5,4 +5,7 @@
     .event-wrapper
       = link_to sanitize(event.name), mweb_events.event_path(event), :class => "event-title"
       %span.event-date
-        = "#{event.start_on.to_formatted_s(:short)} - #{event.end_on.to_formatted_s(:short)}".html_safe
```

Comment:
Wasn't `to_formatted_s` already using a localized string?

We already have the formats on the locales: https://github.com/mconf/mconf-web/blob/4e26136419a92b321c7d396350fb76a61ea6d1cc/config/locales/en/_base.yml#L33
So we shouldn't need to add a new one, unless it's a really specific format used only here.
And I believe `I18n::localize` can also find the formats on their default location in the locales. For example: `I18n::localize(event.start_on, format: :short)`.